R/expectations-matches.R

```diff
@@ -79,11 +83,11 @@ matches <- function(regexp, all = TRUE, ...) {
     matches <- grepl(regexp, char, ...)
     if (length(char) > 1) {
       values <- paste0("Actual values:\n",
-        paste0("* ", encodeString(char), collapse = "\n"))
+                       paste0("* ", encodeString(char), collapse = "\n"))
     } else {
```

Comment:
hmm, okay, I'll revert. You work at RStudio, right? You could try to convince them to do it your way. :)